EIPS/eip-4844.md

```diff
@@ -269,24 +269,25 @@ The actual `data_fee` as calculated via `calc_data_fee` is deducted from the sen
 Blob transactions have two network representations. During transaction gossip responses (`PooledTransactions`), the EIP-2718 `TransactionPayload` of the blob transaction is wrapped to become:
 
 ```
-rlp([blob_tx_payload, blob_kzgs, blobs, blob_kzg_proofs])
+rlp([tx_payload, blobs, commitments, proofs])
 ```
 
 Each of these elements are defined as follows:
 
-- `blob_tx_payload` - standard EIP-2718 blob transaction `TransactionPayload`
-- `blob_kzgs` - list of `KZGCommitment`
-- `blobs` - list of `blob` where `blob` is a list of `BLSFieldElement`
-- `kzg_aggregated_proof` - `KZGProof`
+- `tx_payload` - standard EIP-2718 blob transaction `TransactionPayload`
+- `blobs` - list of `blob` bytes where each `blob` is its `BLSFieldElement` list flattened in `big endian`
+- `commitments` - list of `KZGCommitment` of the corresponding `blobs`
+- `proofs` - list of `KZGProof` of the corresponding `blobs` and `commitments`
 
-The node MUST validate `blob_tx_payload` and verify the wrapped data against it. To do so, ensure that:
+The node MUST validate `tx_payload` and verify the wrapped data against it. To do so, ensure that:
 
-- `blob_tx_payload.blob_versioned_hashes` must not be empty
-- All hashes in `blob_tx_payload.blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
-- There must be at most `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB` total blob commitments in the transaction.
-- There are an equal number of versioned hashes, kzg commitments, and blobs.
-- The KZG commitments hash to the versioned hashes, i.e. `kzg_to_versioned_hash(kzg[i]) == versioned_hash[i]`
-- The KZG commitments match the blob contents. (Note: this can be optimized using `blob_kzg_proofs`, with a proof for a
+- `tx_payload.to` must be set to a valid address
+- `tx_payload.blob_versioned_hashes` must not be empty
+- All hashes in `tx_payload.blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
+- There must be at most `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB` total blob `commitments` in the transaction.
+- There are an equal number of  versioned_hashes, blobs, commitments and proofs.
+- The KZG commitments hash to the versioned hashes, i.e. `kzg_to_versioned_hash(commitments[i]) == tx_payload.blob_versioned_hashes[i]`
+- The KZG `commitments` match the corresponding `blobs` and `proofs`. (Note: this can be optimized using `blob_kzg_proofs`, with a proof for a
```

Comment:
you have 'commitments' here in backquotes but no quotes in the previous line.  Should that line use backquotes as well?  What about the field refs on L288?

Reply:
updated :+1: